docs/source/developer/xkcd_extension_tutorial.rst

```diff
@@ -0,0 +1,6 @@
+  .. _xkcd_extension_tutorial:
+
+We've moved!
```

Comment:
`The extension tutorial has moved!`

I just don't want people to confuse "We", and think that Jupyter or JupyterLab has moved.

Reply:
Haha yes that it is an important thing to clarify.